content/concepts/payment-system-basics/accounts/tickets.md

```diff
@@ -0,0 +1,75 @@
+
+
+## Tickets Are Reserved Sequence Numbers
+
+A Ticket is a record that a sequence number has been set aside to be used later. An account first sends a [TicketCreate transaction][] to set aside one or more sequence numbers as Tickets; this puts a record in the [ledger's state data](ledgers.html), in the form of a [Ticket object][], for each sequence number reserved.
+
+Tickets are numbered using the sequence numbers that were set aside to create them. For example, if your account's current sequence number is 101 and you create 3 Tickets, those Tickets have Ticket Sequence numbers 101, 102, and 103. Doing so increases your account's sequence number to 104.
+
+{{ include_svg("img/ticket-creation.svg", "Diagram: Creating three Tickets") }}
+
+Later, you can send a transaction using a specific Ticket instead of a sequence number; doing so removes the corresponding Ticket from the ledger's state data and does not change your account's normal sequence number. You can also still send transactions using normal sequence numbers without using Tickets. You can use any of your available Tickets in any order at any time, but each Ticket can only be used once.
+
+{{ include_svg("img/ticket-usage.svg", "Diagram: Using Ticket 102.") }}
+
+Continuing the above example, you can send a transaction using sequence number 104 or any of the three Tickets you created. If you send a transaction using Ticket 102, doing so deletes Ticket 102 from the ledger. Your next transaction after that can use sequence number 104, Ticket 101, or Ticket 103.
+
+**Caution:** Each Ticket counts as a separate item for the [owner reserve](reserves.html), so you must set aside 5 XRP for each Ticket. (The XRP becomes available again after you use the Ticket.) This cost can add up quickly if you create a large number of Tickets at once.
+
+As with sequence numbers, sending a transaction uses up the Ticket _if and only if_ the transaction is confirmed by [consensus](consensus.html). However, transactions that fail to do what they were intended to do can still be confirmed by consensus with [`tec`-class result codes](tec-codes.html).
+
+To look up what Tickets an account has available, use the [account_objects method][].
+
+## Limitations
+
+Any account can create and use Tickets on any type of transaction. However, some restrictions apply:
+
+- Each Ticket can only be used once. It is possible to have multiple different candidate transactions
+- Each account cannot have more than 250 Tickets in the ledger at a time. Therefore, you cannot create more than 250 Tickets at a time, either.
+- You _can_ use a Ticket to create more Tickets. If you do, the Ticket you used does not count towards the total number of Tickets you can have at once.
+- Each Ticket counts toward the [owner reserve](reserves.html), so you must set aside 5 XRP for each Ticket you have not used yet. The XRP becomes available for you to use again after the Ticket is used.
```

Comment:
Maybe this is obvious, but I wonder if it is worth mentioning...

> The transaction that creates tickets either succeeds and makes all of the requested tickets or it fails and creates none of them.  This is an important consideration if your account may not contain enough XRP to cover the reserve for all the tickets requested by the transaction.

Just a thought.  Your call.

Reply:
I added a note to this effect to the transaction description; I think details like this are more relevant there than on this concepts page.